### PR TITLE
feature/wasm-ipwindow

### PR DESCRIPTION
**Pull request by Toine-ddt**

### CODE REVIEW

File: src/IpAddress/ScoreConnection.qml [hunk 1]
```diff
@@ -5,7 +5,7 @@
   * - when the button is pressed: a window appears asking for IP address
   */
 
-import QtQuick 2.0
+import QtQuick 2
```
Comment (jcelerier) [inline src/IpAddress/ScoreConnection.qml:8]: weird, maybe 2.12 or 2.15?

File: src/main.qml [hunk 2]
```diff
@@ -12,8 +16,12 @@ Window {
...
-    color: Skin.darkGray
+    color: Skin.green
+
+    Keyboard {
+
+    }
 
     // Creation of the application's skeleton
-    ScoreSkeleton {}
+    //ScoreSkeleton {}
```
Comment (jcelerier) [inline src/main.qml:26]: I guess this means that this PR is a big WIP, no ? :p

File: src/Keyboard.qml [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                }
+                TextField {
+                    id: lowerCaseField
+                    width: parent.width
+                    placeholderText: "Lower case field"
+                    inputMethodHints: Qt.ImhLowercaseOnly
+                    enterKeyAction: EnterKeyAction.Next
+                    onAccepted: phoneNumberField.focus = true
+                }
+                TextField {
```
Comment (jcelerier) [inline src/Keyboard.qml:77]: Maybe take inspiration on this one for the IP input

**Review by jcelerier — CHANGES_REQUESTED** [review]
see comments

**Comment by jcelerier** [comment]
also the title of the PR seems not that much related to the content which adds the virtualkeyboard thing

File: src/main.qml (hunk 2 above)
Comment (Toine-ddt) [reply src/main.qml:26]: Yes... Sorry, I got confused in two PRs :(

File: src/Keyboard.qml (hunk 3 above)
Comment (Toine-ddt) [reply src/Keyboard.qml:77]: ok

File: src/Keyboard.qml [hunk 4]
```diff
@@ -0,0 +1,117 @@
+
```
[the remark this answers is not in the document]
Comment (Toine-ddt) [reply src/Keyboard.qml:1]: ok

File: src/IpAddress/ScoreConnection.qml (hunk 1 above)
Comment (Toine-ddt) [reply src/IpAddress/ScoreConnection.qml:8]: yes

**Comment by Toine-ddt** [comment]
> also the title of the PR seems not that much related to the content which adds the virtualkeyboard thing

Yes... Sorry, I got confused in two PRs :(

**Pull request closed without merging**